core/test.py

```diff
@@ -1,8 +1,37 @@
+from abc import ABCMeta
+
 from unittest import TestCase
 
+from core import app
+from user.models import User
+from blog.models import Article
 from .utils import clean_html
 
 
+class BaseTest(TestCase):
+    __metaclass__ = ABCMeta
+
+    def setUp(self):
+        self.client = app.test_client()
+
+    def tearDown(self):
+        User.objects.delete()
+        Article.objects.delete()
+
+    def login_user(self):
```

Comment:
I see your point. I don't have any cleaner workaround, so it's good for me to keep the `/login/` call for login in.